libs/cli/langchain_cli/integration_template/integration_template/document_loaders.py

```diff
@@ -0,0 +1,78 @@
+"""__ModuleName__ document loader."""
+
+from typing import Iterator
+from langchain_core.document_loaders.base import BaseLoader
+from langchain_core.documents import Document
+
+
+class __ModuleName__Loader(BaseLoader):
+    # TODO: Replace all TODOs in docstring. See example docstring:
+    # https://github.com/langchain-ai/langchain/blob/869523ad728e6b76d77f170cce13925b4ebc3c1e/libs/community/langchain_community/document_loaders/recursive_url_loader.py#L54
+    """
+    __ModuleName__ document loader integration
+    
+    # TODO: Replace with relevant packages, env vars.
+    Setup:
+        Install ``__package_name__`` and set environment variable ``__MODULE_NAME___API_KEY``.
+
+        .. code-block:: bash
+
+            pip install -U __package_name__
+            export __MODULE_NAME___API_KEY="your-api-key"
+
+    # TODO: Replace with relevant init params.
+    Instantiate:
+        .. code-block:: python
+
+            from __module_name__ import __ModuleName__Loader
+
+            loader = __ModuleName__Loader(
+                url = "https://docs.python.org/3.9/",
+                # otherparams = ...
+            )
+
+    Load:
```

Comment:
IMO I think the lazy loading methods are more important for document loaders even -- users should likely favor lazy loading over load even in their own code -- the load functionality is really mostly for notebook envs not production code

Reply:
Sounds good, editing in current commit.